apps/freelance/src/pages/NewJobPosting/JobPostingThirdPage/index.tsx

```diff
@@ -0,0 +1,21 @@
+import { useTranslation } from "react-i18next";
+import { JobPostingContainer } from "@freelance/components";
+import { THIRD_PAGE } from "src/utils/constants/breakpoint";
+import { CREATE_NEW_JOB_SECOND_PAGE } from "utils/constants/links";
+
+export function JobPostingThirdPage() {
+	const { t } = useTranslation();
+
+	const subTitle: string = t("newJobPosting.subTitle");
```

Comment:
why we need it as separate constant?

Reply:
I thought it would make my code more clean, but, ok, I'll pass it into the props.